itests/fevm_test.go

```diff
@@ -1053,3 +1053,25 @@ func TestFEVMErrorParsing(t *testing.T) {
 		})
 	}
 }
+
+func TestEthGetBlockReceipts(t *testing.T) {
+	ctx, cancel, client := kit.SetupFEVMTest(t)
+	defer cancel()
+
+	// Deploy a contract to generate a transaction
+	_, contractAddr := client.EVM().DeployContractFromFilename(ctx, "contracts/SimpleCoin.hex")
+
+	// Get the latest block
+	latestBlock, err := client.EthGetBlockByNumber(ctx, "latest", true)
+	require.NoError(t, err)
+
+	// Get block receipts
+	receipts, err := client.EthGetBlockReceipts(ctx, ethtypes.EthBlockNumberOrHash{BlockHash: &latestBlock.Hash})
```

Comment:
this isn't working because nothing was invoked on "latest", it's probably even a null round

you need to trigger something to happen, then use the tipset that it was submitted on to get the receipts

it's probably easiest just to look for places where `EthGetTransactionReceipt` is currently used and piggy-back that